### PR TITLE
feat(DateRangePicker): add component

### PR DESCRIPTION
**Pull request by PrincAm**
Only DataRangePicker.tsx, styles, prop types and dependencies were updated compared to https://github.com/securityscorecard/design-system/pull/53

### CODE REVIEW

File: src/components/Filters/DateRangePicker/DateRangePicker.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
```
Comment (ajkl2533) [inline src/components/Filters/DateRangePicker/DateRangePicker.tsx:5]: 🤦  😅

File: src/components/Filters/FilterRow/FilterRow.tsx [hunk 2]
```diff
@@ -123,7 +124,11 @@ const FilterRow: React.FC<FilterRowProps> = ({
   };
 
   const handleInputChange = (event) => {
-    onInputChange(event.target.value, index);
+    if (event.target) {
```
Comment (ajkl2533) [inline src/components/Filters/FilterRow/FilterRow.tsx:127]: Why do we need to split this in condition?

**Review by ajkl2533 — APPROVED** [review]
I don't see anything wrong :)

File: src/components/Filters/FilterRow/FilterRow.tsx (hunk 2 above)
Comment (PrincAm) [reply src/components/Filters/FilterRow/FilterRow.tsx:127]: It is generic handler
- if input component is `Input` then event is Event type
- if input component is DateRangePicker then event is object `{startDate, endDate}`
- if input component is Select then event is object `{value, label}` - not yet implemented
- if input component is SingleDatePicker then event is date - not yet implemented

Maybe event is misleading name now, I can change it. Also the condition will change when I will add other components.

File: src/components/Filters/DateRangePicker/DateRangePicker.tsx (hunk 1 above)
Comment (PrincAm) [reply src/components/Filters/DateRangePicker/DateRangePicker.tsx:5]: Hm, what can we do. Now I believe there is not better option than `react-datepicker` 😄

File: src/components/Filters/FilterRow/FilterRow.tsx (hunk 2 above)
Comment (ajkl2533) [reply src/components/Filters/FilterRow/FilterRow.tsx:127]: ah ok I was misled by the name

Comment (PrincAm) [reply src/components/Filters/FilterRow/FilterRow.tsx:127]: Ok, I will rename it in next PR

**Pull request merged**